src/touchaction.js

```diff
@@ -83,6 +83,19 @@ TouchAction.prototype = {
         var hasPanY = inStr(actions, TOUCH_ACTION_PAN_Y);
         var hasPanX = inStr(actions, TOUCH_ACTION_PAN_X);
 
+
+        if (hasNone) {
+            //do not prevent defaults if this is a tap gesture
+
+            var isTapPointer = input.pointers.length === 1;
+            var isTapMovement = input.distance === 0;
```

Comment:
I think finger can shake on tap or press, so I suggest to write it as 
`var isTapMovement = input.distance < 5;`
5 — is threshold here